### PR TITLE
July newsletter

### PR DESCRIPTION
**Pull request by LKedward**
Hi all - I've started the initial draft for this Month's newletter.

- [x] add stdlib updates
- [x] add fpm updates
- [x] add fortran-lang.org updates
- [x] add monthly call link
- [ ] ~add LLVM / classic flang updates (@AlexisPerry)~
- [x] add LFortran updates (@certik)
- [x] update contributor data (@LKedward)

As usual, please add your name to the authors list if you add, edit, or review content.

### CODE REVIEW

**Comment by intrigus-lgtm** [comment]
#build_preview

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+<ul id="page-nav"></ul>
+
+## fortran-lang.org
+
+This month we've had several updates to the website:
+
+* [#276](https://github.com/fortran-lang/fortran-lang.org/pull/276):
+  Add LATTE tight-binding molecular dynamics code to package index                                                               
+* [#275](https://github.com/fortran-lang/fortran-lang.org/pull/275):
+  Add crest program to package index
```
Comment (zmoon) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:21]:
```suggestion
  Add crest program to package index
* [#255](https://github.com/fortran-lang/fortran-lang.org/pull/255):
  Quickstart edits
```

**Comment by LKedward** [comment]
@certik & @AlexisPerry do you have any updates you want to add to the newsletter?

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  Add the SNaC package to package index
+* [#272](https://github.com/fortran-lang/fortran-lang.org/pull/272):
+  Add QUICK to package index
+
+Ongoing work:
+
+* [#277](https://github.com/fortran-lang/fortran-lang.org/pull/277) (WIP):
+  Add projects for Fortran-lua interfacing to package index                                                                
+* [#274](https://github.com/fortran-lang/fortran-lang.org/pull/274) (WIP):
+  Update package_index.yml
```
Comment (awvwgk) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:34]:
```suggestion
  Add convert_FORTRAN_case formatter to package index
```

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+[Let us know](https://github.com/fortran-lang/fortran-lang.org/issues) if you have any suggestions for the website and its content.
+We welcome any new contributors to the website and the tutorials page in particular - see the [contributor guide](https://github.com/fortran-lang/fortran-lang.org/blob/master/CONTRIBUTING.md) for how to get started.
+
+## Fortran Standard Library
+
+Here's what's new in `stdlib`:
+
+* [#439](https://github.com/fortran-lang/stdlib/pull/439) (WIP):
+  Reduce time spent on sorting tests
+* [#440](https://github.com/fortran-lang/stdlib/pull/440) (WIP):
```
Comment (awvwgk) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:49]:
```suggestion
* [#439](https://github.com/fortran-lang/stdlib/pull/439):
  Reduce time spent on sorting tests
* [#440](https://github.com/fortran-lang/stdlib/pull/440):
```

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+  Allow modification of install directory for module files
+* [#430](https://github.com/fortran-lang/stdlib/pull/430):
+  Remove support for GCC 7 and 8
+* [#424](https://github.com/fortran-lang/stdlib/pull/424):
+  Add separate logical kind parameters
+
+Work in progress:
+
+* [#445](https://github.com/fortran-lang/stdlib/pull/445) (WIP):
+  Add `disp`(display your data) & `format_string`(format other type to string)
```
Comment (awvwgk) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:67]:
```suggestion
* [#445](https://github.com/fortran-lang/stdlib/pull/445) (WIP):
  Add `disp` function to display your data
* [#444](https://github.com/fortran-lang/stdlib/pull/444) (WIP):
  Add `format_string` to format other type to string
```

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title: "Fortran newsletter: July 2021"
+category: newsletter
+author: Laurence Kedward
```
Comment (LKedward) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:5]:
```suggestion
author: Laurence Kedward, Sebastian Ehlert
```

**Review by awvwgk — APPROVED** [review]
Thanks for putting this together.

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+
+
+
+## Events
+
+* We had our 13th Fortran Monthly call on June 15.
+You can watch the recording below:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YRVLAlQpE5g" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
```
Comment (milancurcic) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:178]:
```suggestion

* Joint J3/WG5 (Fortran Standards Committees) meeting was held virtually from June 21-30 (Mondays and Wednesdays only). You can find all the papers that were discussed [here](https://j3-fortran.org/doc/meeting/224). Highlights from the meeting:
  - Conditional expressions syntax for Fortran 202X ([paper](https://j3-fortran.org/doc/year/21/21-157r2.txt)).
  - Protected components specifications and syntax for Fortran 202X ([paper](https://j3-fortran.org/doc/year/21/21-168.txt)).
  - The generics feature planned for Fortran 202Y was discussed at depth ([paper](https://j3-fortran.org/doc/year/21/21-144r4.txt)).
  - Jeff Hammond (NVidia Corporation) is the new J3 member as a voting alternate to Bryce Adelstein-Lelbach.
  - Target year for Fortran 202X is 2023, subject to change.

* FortranCon 2021 will be held virtually from September 23-24, 2021. For more information, visit the [FortranCon website](https://tcevents.chem.uzh.ch/event/14/).

```

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+* [#420](https://github.com/fortran-lang/stdlib/pull/420) (WIP):
+  First implementation of real-valued linspace.
+* [#363](https://github.com/fortran-lang/stdlib/pull/363) (WIP):
+  Sorting string's characters according to their ASCII values
+* [#353](https://github.com/fortran-lang/stdlib/pull/353) (WIP):
+  Initial checkin for a module for tolerant comparison of reals
+* [#333](https://github.com/fortran-lang/stdlib/pull/333) (WIP):
+  Provide abstract base class for a string object
+* [#313](https://github.com/fortran-lang/stdlib/pull/313) (WIP):
+  Legendre polynomials and gaussian quadrature
```
Comment (milancurcic) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:89]: Merged and moved up

```suggestion
```

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+* [#201](https://github.com/fortran-lang/fortran-lang.org/pull/201) (WIP):
+  Draft: Internationalization for fortran-lang    
+
+[Let us know](https://github.com/fortran-lang/fortran-lang.org/issues) if you have any suggestions for the website and its content.
+We welcome any new contributors to the website and the tutorials page in particular - see the [contributor guide](https://github.com/fortran-lang/fortran-lang.org/blob/master/CONTRIBUTING.md) for how to get started.
+
+## Fortran Standard Library
+
+Here's what's new in `stdlib`:
+
```
Comment (milancurcic) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:46]:
```suggestion

* [#313](https://github.com/fortran-lang/stdlib/pull/313):
  Legendre polynomials and gaussian quadrature
* [#432](https://github.com/fortran-lang/stdlib/pull/432):
  Outer product
```

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+* [#444](https://github.com/fortran-lang/stdlib/pull/444) (WIP):
+  Add `format_string` to format other type to string
+* [#441](https://github.com/fortran-lang/stdlib/pull/441) (WIP):
+  Implement pad function
+* [#437](https://github.com/fortran-lang/stdlib/pull/437) (WIP):
+[FPM] add fpm support
+* [#436](https://github.com/fortran-lang/stdlib/pull/436) (WIP):
+  Implement low-level replace_all function
+* [#432](https://github.com/fortran-lang/stdlib/pull/432) (WIP):
+  Outer product
```
Comment (milancurcic) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:77]: Merged and moved up

```suggestion
```

File: _posts/2021-07-01-Fortran-Newsletter-July-2021.md [hunk 10]
```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: "Fortran newsletter: July 2021"
+category: newsletter
+author: Laurence Kedward, Sebastian Ehlert
```
Comment (milancurcic) [inline _posts/2021-07-01-Fortran-Newsletter-July-2021.md:5]:
```suggestion
author: Laurence Kedward, Sebastian Ehlert, Milan Curcic
```

**Review by milancurcic — APPROVED** [review]
Thank you!

**Comment by certik** [comment]
Yes, I am working on it now.

**Review by certik — APPROVED** [review]
Ok, I am done. I merged @milancurcic's changes, I added LFortran's changes.

**Comment by LKedward** [comment]
Thanks everyone!

**Comment by awvwgk** [comment]
#delete_preview

**Pull request merged**